OG format governace

```diff
@@ -17,20 +17,20 @@ The format will need to grow and evolve with time while being managed effectivel
 
 1. The team will analyze the issue formally. If the issue is not detailed enough or if there is no example associated with the issue, the issue will be rejected. If the issue is accepted for review, then the team will study it and provide an answer. The issue can lead to a modification of the format or not. If there is no consensus on the solution, the suggested solution will be rejected and the format will remain unchanged. However, a new branch can be created to demonstrate the benefit of the solution. The same issue can be submitted again with a demo.
 
-2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for appraval. Stakeholers will be noftifed of the change at this point.
+2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for approval. Stakeholers will be noftifed of the change at this point.
 
```

Comment:
How do we guaranty that "Stakeholers will be noftifed of the change" ?

Reply:
I think we need to leverage the github/open science framework, may need some expert help from @soerenthomsen  on how to achieve this

Reply:
If stakeholders are individuals/groups who have contributed to the repo, they will be notified of new releases, pull requests etc.It's up to the individual to decide what github notifications they recieve. Alternatively, if you know who the stakeholders are that you wish to notify, you can @ them like @vturpin to attract their attention to a change.